Speedykom/Regional-Pandemic-Analytics
# Revert/remove vault service

## Description ##
This PR aims to revert back the project structure and functionalities to no longer support Vault service

## Changes ##
- removed all config files of Vault]
- added old env files]
- removed script for generating env files]
- changed pipeline execution to no longer start vault before starting the rest of containers
- modified README for clear instructions
- removed vault service from docker-compose
- removed env files from gitignore]

## Testing Instructions ##
- Start the project according to the corresponding env using make command
- Verify that the containers are healthy and project is running 
- Verify all pipeline jobs are successful
